src/lib/components/data-graphic/utils.ts

```diff
@@ -73,3 +74,25 @@ export function areaFactory(args: LineGeneratorArguments) {
       .curve(curves[args.curve] || curveLinear)
       .defined(args.pathDefined || pathDoesNotDropToZero(yAccessor));
 }
+
+/**
+ * Return a list of ticks to be represented on the
+ * axis or grid depending on axis-side, it's length and
+ * the data type of field
+ */
+export function getTicks(
+  xOrY: string,
+  scale: GraphicScale,
+  axisLength: number,
+  isDate: boolean
+) {
+  const tickCount = ~~(axisLength / (xOrY === "x" ? 150 : 50));
+  let ticks = scale.ticks(tickCount);
+
+  if (ticks.length <= 1) {
+    if (isDate) ticks = scale.domain();
```

Comment:
```suggestion
    if (isDate) ticks = [...ticks, ...scale.domain()] as Date[];
```

This one change will make sure that we get at least three ticks in the metrics dash
<img width="679" alt="Screen Shot 2022-07-27 at 4 44 03 PM" src="https://user-images.githubusercontent.com/95735/181390912-5756f134-045c-4125-a234-9c607a3979fd.png">